test/e2e/specs/editor/various/publish-panel.spec.js

```diff
@@ -57,4 +57,60 @@ test.describe( 'Post publish panel', () => {
 			)
 		).toBeFocused();
 	} );
+
+	test( 'should move focus to the publish button in the panel', async ( {
+		editor,
+		page,
+	} ) => {
+		await editor.canvas
+			.getByRole( 'textbox', { name: 'Add title' } )
+			.fill( 'Test Post' );
+		await page
+			.getByRole( 'region', 'Editor top bar' )
+			.getByRole( 'button', { name: 'Publish', exact: true } )
+			.click();
+
+		await expect(
+			page
+				.getByRole( 'region', { name: 'Editor publish' } )
+				.locator( ':focus' )
+		).toHaveText( 'Publish' );
```

Comment:
Could we instead rewrite this into `toBeFocused` to better match the description?

Reply:
Don't have a strong opinion, but the original test was checking the `activeElement`.